src/DAVerifier.sol

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.4;
```

Comment:
Bump to v0.8.19 in here and other new files.

Reply:
https://github.com/celestiaorg/quantum-gravity-bridge/pull/138/commits/211fc70379190996ab4001393bc73b49d784ac7b